### PR TITLE
Unflow dev

### PR DESCRIPTION
**Pull request by NeedsMorePie**

### CODE REVIEW

File: pwcnet/losses/loss.py [hunk 1]
```diff
@@ -50,9 +53,95 @@ def create_multi_level_loss(expected_flow, flows, flow_scaling, flow_layer_loss_
...
+    total_loss = tf.add_n(layer_losses, name='total_loss')
     return total_loss, layer_losses
+
+
+def create_multi_level_unflow_loss(image_a, image_b, forward_flows, backward_flows, flow_scaling,
+                                   flow_layer_loss_weights=None, layer_patch_distances=None, loss_weights=None):
+    """
+    :param image_a: Tensor of shape [B, H, W, 3].
+    :param image_b: Tensor of shape [B, H, W, 3].
+    :param forward_flows: List of tensors of shape [B, H, W, 2]. Flows at different resolutions.
```
Comment (boxofpasta) [inline pwcnet/losses/loss.py]: Mention how these resolutions should be ordered for the default flow_layer_loss_weights somewhere.

File: pwcnet/losses/loss.py [hunk 2]
```diff
@@ -50,9 +53,95 @@ def create_multi_level_loss(expected_flow, flows, flow_scaling, flow_layer_loss_
...
+    :param backward_flows: List of tensors of shape [B, H, W, 2]. Flows at different resolutions.
+    :param flow_scaling: Float. Scales the expected_flow before comparing it to the flows.
```
Comment (boxofpasta) [inline pwcnet/losses/loss.py]: Needs to be clarified, there's no expected_flow that's being passed in.

File: pwcnet/losses/loss.py (hunk 1 above)
Comment (NeedsMorePie) [reply pwcnet/losses/loss.py]: Done.

File: pwcnet/losses/loss.py (hunk 2 above)
Comment (NeedsMorePie) [reply pwcnet/losses/loss.py]: Done.
